andela/learning-map-front
# #150915443 Show snackbar on toogle bookmark

**What does this PR do?**

Displays a snackbar at the bottom of the page when a contribution's bookmark icon is toggled.

**Description of Task to be completed?**

- Show snackbar when a user bookmarks a contribution
- Show snackbar when a user unbookmarks a contribution
- adjust height of sidebar to fit automatically adjust to different screen sizes
- Rearrange display of bookmarks on sidebar to show from latest to earliest

**How should this be manually tested?**

- Run yarn install
- Run yarn start
- Log in with your andela email
- When you're successfully logged in, you should see the different contributions
- Bookmark a contribution with the bookmark icon
- A snack bar should appear at the bottom describing the action you just performed `Your bookmark has been added` or `Your bookmark has been removed`

**Any background context you want to provide?**
N/A

**What are the relevant pivotal tracker stories?**

[#150915443](https://www.pivotaltracker.com/story/show/150915443)

**Screenshots (if appropriate)**

<img width="1279" alt="screen shot 2017-09-13 at 20 13 22" src="https://user-images.githubusercontent.com/28440972/30390840-11368262-98c0-11e7-9d2f-0e604294e733.png">

<img width="1278" alt="screen shot 2017-09-13 at 20 13 30" src="https://user-images.githubusercontent.com/28440972/30390842-12689076-98c0-11e7-95a8-7a1ba3e90413.png">
